Integrations/integration-PostgreSQL.yml

```diff
@@ -71,17 +71,27 @@ script:
     if demisto.command() == 'pgsql-query':
         conn = pgsqlConnect()
         cur = None
+        rows = []
         try:
             cur = conn.cursor()
             cur.execute(demisto.args()['query'])
-            rows = cur.fetchall()
-            colnames = [desc[0] for desc in cur.description]
+            # try to fetch rows
+            try:
+                colnames = [desc[0] for desc in cur.description]
+                rows = cur.fetchall()
+            except:
```

Comment:
catch the error and ignore only errors which return `no results to fetch`
if the command failed because other reason, then raise the original error.